src/pages/home/report/ReportActionsView.js

```diff
@@ -46,8 +62,15 @@ class ReportActionsView extends React.Component {
         this.scrollToListBottom = this.scrollToListBottom.bind(this);
         this.recordMaxAction = this.recordMaxAction.bind(this);
         this.onVisibilityChange = this.onVisibilityChange.bind(this);
-        this.sortedReportActions = this.updateSortedReportActions();
+        this.setUpUnreadActionIndicator = this.setUpUnreadActionIndicator.bind(this);
+        this.updateSortedReportActions = this.updateSortedReportActions.bind(this);
+
+        this.sortedReportActions = [];
         this.timers = [];
+        this.unreadActionCount = 0;
+        this.shouldShowNewActionIndicator = true;
+        this.unreadIndicatorOpacity = new Animated.Value(1);
+        this.unreadIndicatorTimer = null;
```

Comment:
Also here, do we have to use a specific variable instead of pushing the timer into the `timers` array? 

I only see us really modifying and removing it here https://github.com/Expensify/Expensify.cash/blob/5a145d46fdbc42573b2e49ae15e2d64e59d6a8b5/src/pages/home/report/ReportActionsView.js#L147

When a few lines below we do the same thing with `timers`.

Reply:
We have not, my bad.